oxen-rust/cli-test/README.md

```
@@ -1,13 +1,13 @@
 
-## Setup 
+## Setup
 
-1. From the `cli-test` folder, run `bundle install` to install the required gems. 
+1. From the `cli-test` folder, run `bundle install` to install the required gems.
+
+2. (Currently only needed for the performance test) Create a `.env` file in the `cli-test` folder containing the credentials for the test user (@test):
 
-2. Create a `.env` file in the `cli-test` folder containing the credentials for the test user (@test): 
-    
     ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)